contracts/networks/local.json

```diff
@@ -0,0 +1,18 @@
+{
+    "parameters": {},
+    "addresses": {
+        "precompileCostEstimator": "",
+        "verificationGateway": "0xCf7Ed3AccA5a467e9e704C703E8D87F634fB0Fc9",
+        "blsLibrary": "0x9fE46736679d2D9a65F0992F2272dE9f3c7fa6e0",
+        "blsExpander": "0x5FC8d32690cc91D4c39d9d3abcBD16989F875707",
+        "testToken": "",
+        "rewardToken": ""
+    },
+    "auxiliary": {
+        "chainid": 1337,
+        "domain": "0x0054159611832e24cdd64c6a133e71d373c5f8553dde6c762e6bffe707ad83cc",
+        "genesisBlock": 0,
+        "deployedBy": "TODO fill in",
+        "version": "c1d1f197ba9b68284cbc4ec145e286a4e84c0eaf"
```

Comment:
When the local deploy script is ironed out when can have these all be generated and treat this like `.env` files.